src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -305,6 +307,12 @@ public ScenarioLoader(GradleBuildConfigurationReader gradleBuildConfigurationRea
         return definitions;
     }
 
+    private static StudioGradleScenarioDefinition newStudioGradleScenarioDefinition(GradleScenarioDefinition gradleScenarioDefinition, Config scenario) {
+        Config androidStudioSync = scenario.getConfig(ANDROID_STUDIO_SYNC);
+        List<String> studioJvmArgs = ConfigUtil.strings(androidStudioSync, ANDROID_STUDIO_JVM_ARGS, Collections.emptyList());
```

Comment:
Should the default be `["-Xms256m", "-Xmx4096m"]` and not the empty list?

Reply:
I set it to `["-Xms256m", "-Xmx2048m"]` first as we had before, but then I removed them. I am sure, maybe I can add them back, not sure what is better here

Reply:
For me, the basic question is if Android studio works without any jvmArgs. If it does, then I suppose leaving them out is fine. If it doesn't, then we should use some sensible default.

Reply:
Without any jvm args it works. But it sets Xmx to default Java xmx. On my machine that is 16GB (1/4 of all available memory).

I set default to  `["-Xms256m", "-Xmx4096m"]` I think it's sensible and if anyone has issue with it, it can override it.